esp/templates/program/modules/finaidapprovemodule/finaid.html

```diff
@@ -58,7 +58,7 @@ <h1>Financial Aid Requests for {{ program.niceName }}</h1>
             <i align="center" class="glyphicon glyphicon-minus-sign" style="color:firebrick; width:100%; font-size:1.5em;" />
             {% endif %}
           </td>
-          <td align="center" style="max-width: 70px; overflow-wrap:break-word;">{{ req.user.name }}</td>
+          <td align="center" style="max-width: 70px; overflow-wrap:break-word;"><a href="/manage/userview?username={{ req.user.username|urlencode }}">{{ req.user.name }}</a></td>
```

Comment:
same encoding question here on req.user.name

Reply:
Probably this one is ok since Django's autoescaping mechanism takes care of escaping inside {{ }}.